src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -828,6 +828,12 @@ public void test_clean() throws Exception {
         w.touch(fileName2);
         w.touch(fileName, "new content");
 
+        // JGit does not support exclude patterns        
+        if (w.igit() instanceof CliGitAPIImpl)
+        {
+        	((CliGitAPIImpl)w.igit()).clean_().excludePatterns(Collections.singletonList(fileName)).execute();
```

Comment:
Is there a specific reason you didn't use w.cgit() instead of casting the w.igit() return value to a CliGitAPIImpl?